chef/cookbooks/provisioner/recipes/update_nodes.rb

```diff
@@ -290,6 +290,7 @@ def find_node_boot_mac_addresses(node, admin_data_net)
           append << "ifcfg=dhcp4 netwait=60"
           append << "squash=0" # workaround bsc#962397
           append << "autoupgrade=1" if mnode[:state] == "os-upgrading"
+          append << "self_update"
```

Comment:
I added settings to `autoyast.xml` so that failures in self update don't mean failure anymore. AutoYaST will log those failure but continue with the installation process.